bazel-java-deps.bzl

```diff
@@ -104,8 +104,8 @@ def install_java_deps():
             "org.junit.jupiter:junit-jupiter-engine:5.0.0",
             "org.junit.platform:junit-platform-engine:1.0.0",
             "org.junit.platform:junit-platform-runner:1.0.0",
-            "org.mockito:mockito-core:2.25.0",
-            "org.mockito:mockito-inline:2.25.0",
+            "org.mockito:mockito-core:3.2.0",
```

Comment:
@miklos-da think you have to rerun `bazel run @unpinned_maven//:pin` and check in the gen'd file from that command whenever you change a version dependency in `bazel-java-deps.bzl`